Don't build a shared libclang.so...
I don't want to make flang anything but a single static binary.

Index: tools/libclang/CMakeLists.txt
--- tools/libclang/CMakeLists.txt.orig
+++ tools/libclang/CMakeLists.txt
@@ -74,7 +74,7 @@ if(MSVC)
   set(LLVM_EXPORTED_SYMBOL_FILE)
 endif()
 
-if( LLVM_ENABLE_PIC )
+if( 0 )
   set(ENABLE_SHARED SHARED)
 endif()
 
